src/Microsoft.Framework.Runtime.Interfaces/Compilation/ICompilationProject.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Runtime.Versioning;
+
+namespace Microsoft.Framework.Runtime.Compilation
+{
+    public interface ICompilationProject
```

Comment:
I think we'll end up nuking this interface when we expose project properties as a dictionary. This will end up being a straight POCO in .Interfaces.